### ISSUE TITLE
Seems not suitable for the bounding box mask?

### ISSUE THREAD

**Issue by xiao10ma** [TencentARC/VideoPainter#38]
I replace the masked_video with a street video masked with the dynamic bounding box and the mask with the bounding box. However, there's still many blank wholes in the generated video:

https://github.com/user-attachments/assets/d4d17862-7162-450e-bb96-7fc3c35c58d5

Do you have any ideas about the reasons?

ps: the reason why the second masked_image is gray, I think is because of the video code rate. When I save one masked_image, it's black.

**Comment by yxbian23** [comment]
> I replace the masked_video with a street video masked with the dynamic bounding box and the mask with the bounding box. However, there's still many blank wholes in the generated video:
> 
>  541_fps8.mp4 
> Do you have any ideas about the reasons?
> 
> ps: the reason why the second masked_image is gray, I think is because of the video code rate. When I save one masked_image, it's black.

In my opinion, this is because we didn't add multi-bounding-box masks in the training process. Our main training data is about segmentation-based masks. 

But I think this can be simply improved by fine-tuning VideoPainter with multi-bounding-box masks. You can refer issue #34 for more details~

**Comment by xiao10ma** [comment]
Cool, thank you!

**Comment by DanahYatim** [comment]
Hi, did you include multi-segmentation-based masks in the training process? I'm getting a similar artifact with this scenario
